main.rb

```diff
@@ -0,0 +1,101 @@
+require './app'
+
+@app = App.new
+
+OPTIONS = {
+  1 => :list_all_books,
+  2 => :list_all_people,
+  3 => :create_person,
+  4 => :create_book,
+  5 => :create_rental,
+  6 => :list_rentals_for_person,
+  7 => :exit_program
+}.freeze
+
+def main
+  display_menu
+
+  loop do
+    choice = prompt_choice
+    handle_choice(choice)
+    break if choice == 7
+  end
+end
+
+def display_menu
+  puts 'Please select an option:'
+  puts '1. List all books'
```

Comment:
- [ ] I'm getting an erro if I want to display List of books. That is not a good practice. I suggest you to fix up the problem.